src/FlexVotingClient.sol

```diff
@@ -48,7 +48,7 @@ import {IVotingToken} from "./interfaces/IVotingToken.sol";
 /// of the rest.
 abstract contract FlexVotingClient {
   using SafeCast for uint256;
-  using Checkpoints for Checkpoints.History;
+  using Checkpoints for Checkpoints.Trace224;
```

Comment:
Should this be a 208?

Reply:
If you want to allign with ERC-5805, then yes

Reply:
Switched to Trace208 in https://github.com/ScopeLift/flexible-voting/pull/66/commits/a01bf77a1f10d12b57f3a29ee07bd600ddbde973